client/src/actions/authActions.js

```diff
@@ -1,8 +1,8 @@
-import axios from "axios";
-import { Toast } from "antd-mobile";
+import { AUTH_LOGIN, AUTH_SIGNUP } from "../constants/action-types";
 
 import { GET_ERRORS } from "./types";
-import { AUTH_LOGIN, AUTH_SIGNUP } from "../constants/action-types";
+import { Toast } from "antd-mobile";
+import axios from "axios";
```

Comment:
I wonder how this grouping works, there is also an ESLint rule for grouping imports and I think it's usually something like 
- first hte external dependencies
- then the imports from your own local codebase
and then alphabetically, first by import path and then by import name

Reply:
I am going to say something you will not like, I have never used ESLint before and I thought ESLint is just a conversion for writing clean code but now I realize is way beyond it. I did all the sort using just a vscode plugin. So sorry I wasted your and @anniegiang time for making confusing. I am going to make changes while I dig more into ESLint rules.

Reply:
No worries! Thanks for starting on this nevertheless.

Reply:
@TenNga heh... what makes you think I wouldn't like this? I will try to stay objective :p 
But yeah good idea to learn some more about ESLint , we are also using "Prettier" which is kind of related, but Prettier focuses more on code style and formatting... 
Good to know that you have used a VSCode plugin, it should be configured in the ESLint rules so it also works from command line, and this same command also runs with the pre-commit hook which is configured via "husky" in package.json so all these things will pick up the ESlint esttings and use plugins if we configure it like that